src/util/kiltDid.ts

```diff
@@ -0,0 +1,31 @@
+import type { KeyringPair } from '@polkadot/keyring/types'
+
+import { ApiPromise, WsProvider } from '@polkadot/api'
+
+import '@kiltprotocol/augment-api'
+import { typesBundle } from '@kiltprotocol/type-definitions'
+import { chainProperties, network } from '~src/global/networkConstants'
+
+export async function getKiltDidName(
+  lookupAccountAddress: KeyringPair['address']
+): Promise<string | null> {
+  const api = await ApiPromise.create({
```

Comment:
api is already initialised and available using ApiContext, we shoudln't need to initialise it again for kilt